lib/oeqa/selftest/cases/updater_qemux86_64.py

```diff
@@ -395,7 +395,7 @@ def setUpLocal(self):
         self.append_config('IMAGE_INSTALL_append += " aktualizr-resource-control "')
         self.append_config('RESOURCE_CPU_WEIGHT_pn-aktualizr = "1000"')
         self.append_config('RESOURCE_MEMORY_HIGH_pn-aktualizr = "50M"')
-        self.append_config('RESOURCE_MEMORY_MAX_pn-aktualizr = "1M"')
+        #self.append_config('RESOURCE_MEMORY_MAX_pn-aktualizr = "1M"')
```

Comment:
Is it intended?

Reply:
D'oh! Nope. Good catch. I'm glad I asked you to re-review. I was double-checking that it failed as expected. Anyway, fixed now.